server/models/Availability.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: 'User',
+    },
+    petSitterId: {
```

Comment:
I don't think we need two references to the User model, either we can use this field or the one above it